notebooks/templates/Adviser_justification_classification_TEMPLATE.ipynb

```diff
@@ -0,0 +1,1376 @@
+{
```

Comment:
<p>Do we need that cell here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thoth-station/notebooks/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>We can lave it here, if someone want to send the metrics from this notebook beside analyzing results, it's gonna be part of a consumer of kafka messages later</p>